packages/client/components/ScheduleDialog.tsx

```diff
@@ -0,0 +1,210 @@
+                id
+              }
+              cloudProvider {
+                id
+                clientId
+              }
+            }
+          }
+        }
+        ...GcalModal_team
+        ...GcalSettings_team
+      }
+    `,
+    teamRef
+  )
+
+  const {id: teamId, viewerTeamMember} = team
+  const {gcal} = viewerTeamMember?.integrations ?? {}
+
+  const atmosphere = useAtmosphere()
+  const {fields, onChange} = useForm({
+    title: {
+      getDefault: () => ''
+    }
+  })
+  const title = fields.title.value
+  const titleErr = fields.title.error
+
+  const onNameChange = (event: ChangeEvent<HTMLInputElement>) => {
+    if (titleErr) {
+      fields.title.setError('')
+    }
+    onChange(event)
+  }
+
+  const onOpenGcalChange = (open: boolean) => {
+    setHasGcalEvent(open)
+  }
+
+  const onOpenRecurrenceChange = (open: boolean) => {
+    setHasRecurrence(open)
+  }
+
+  const handleSubmit = () => {
+    const title = fields.title.value || placeholder
+    const titleRes = validateTitle(title)
+    if (titleRes.error) {
+      fields.title.setError(titleRes.error)
+      return
+    }
+
+    const gcalEventInput = hasGcalEvent
+      ? {
+          title,
+          startTimestamp: gcalInput.start.unix(),
+          endTimestamp: gcalInput.end.unix(),
+          timeZone: Intl.DateTimeFormat().resolvedOptions().timeZone,
+          invitees: gcalInput.invitees,
+          videoType: gcalInput.videoType ?? undefined
+        }
+      : undefined
+    props.onStartActivity(gcalEventInput, hasRecurrence && rrule ? {rrule} : undefined)
+  }
+
+  const authGCal = () => {
+    if (!gcal?.cloudProvider) {
+      return
+    }
+    const {clientId, id: providerId} = gcal.cloudProvider
+    GcalClientManager.openOAuth(atmosphere, providerId, clientId, teamId, mutationProps)
+
+    SendClientSideEvent(atmosphere, 'Schedule meeting add gcal clicked', {
+      teamId: teamId,
+      service: 'gcal'
+    })
+  }
+
+  return (
+    <div className='space-y-4 p-4'>
+      <div className='text-lg font-semibold leading-none'>Schedule Your Meeting</div>
+      <div className='text-sm text-slate-800'>
+        Create a recurring meeting series or add the meeting to your calendar.
+      </div>
+      <div className='flex flex-col'>
+        <input
+          className='form-input rounded border border-solid border-slate-500 p-2 font-sans text-base hover:border-slate-600 focus:border-slate-600 focus:outline-none focus:ring-1 focus:ring-slate-600'
+          type='text'
+          name='title'
+          placeholder={placeholder}
+          value={title}
+          onChange={onNameChange}
+          min={1}
+          max={50}
+        />
+        {titleErr && <StyledError>{titleErr}</StyledError>}
+      </div>
+      {gcal?.cloudProvider &&
+        (gcal?.auth ? (
+          <Collapsible.Root
+            className='flex flex-col rounded border border-slate-500'
+            open={hasGcalEvent}
+            onOpenChange={onOpenGcalChange}
```

Comment:
-1 after integrating with gcal, I think the Calendar Invite section should automatically open. After I integrated with GCal, it wasn't clear to me that I could edit the settings